CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
+  * The Track method now dispatches its conversion event _unconditionally_, without first determining whether the user is targeted by a known experiment that uses the event. This may increase outbound network traffic.
+  * In Optimizely results, conversion events sent by 3.0 SDKs are automatically attributed to variations that the user has previously seen, as long as our backend has actually received the impression events for those variations.
+  * Altogether, this allows you to track conversion events and attribute them to variations even when you don't know all of a user's attribute values, and even if the user's attribute values or the experiment's configuration have changed such that the user is no longer affected by the experiment. As a result, **you may observe an increase in the conversion rate for previously-instrumented events.** If that is undesirable, you can reset the results of previously-running experiments after upgrading to the 3.0 SDK.
+  * This will also allow you to attribute events to variations from other Optimizely projects in your account, even though those experiments don't appear in the same datafile.
+  * Note that for results segmentation in Optimizely results, the user attribute values from one event are automatically applied to all other events in the same session, as long as the events in question were actually received by our backend. This behavior was already in place and is not affected by the 3.0 release.
+* Support for all types of attribute values, not just strings.
+  * All values are passed through to notification listeners.
+  * Strings, booleans, and valid numbers are passed to the event dispatcher and can be used for Optimizely results segmentation. A valid number is a finite number in the inclusive range [-2⁵³, 2⁵³]. 
+  * Strings, booleans, and valid numbers are relevant for audience conditions.
+* Support for additional matchers in audience conditions:
+  * An `exists` matcher that passes if the user has a non-null value for the targeted user attribute and fails otherwise.
+  * A `substring` matcher that resolves if the user has a string value for the targeted attribute.
+  * `gt` (greater than) and `lt` (less than) matchers that resolve if the user has a valid number value for the targeted attribute. A valid number is a finite number in the inclusive range [-2⁵³, 2⁵³].
+  * The original (`exact`) matcher can now be used to target booleans and valid numbers, not just strings.
+* Support for A/B tests, feature tests, and feature rollouts whose audiences are combined using `"and"` and `"not"` operators, not just the `"or"` operator.
+* Datafile-version compatibility check: The SDK will remain uninitialized (i.e., will gracefully fail to activate experiments and features) if given a datafile version greater than 4.
+* Updated Pull Request template and commit message guidelines.
+* Only remove old variation assignments from the user profile service when there are >= 100 persisted assignments.
```

Comment:
@thomaszurkan-optimizely says this actually went out in 2.1.6. Not sure where we should document this 😕 

@jaeopt @thomaszurkan-optimizely @mikeng13 what do you think? We could add this under 2.1.6 in the `master` branch's changelog, but that wouldn't update the changelog in the `v2.1.6` tag.

Reply:
Okay, on further thought, I would be fine with
1. Updating the 2.1.6 section in the master branch's changelog
2. Updating the description of the 2.1.6 release in GitHub
3. *Not* updating the v2.1.6 to fix its changelog